yt_dlp/extractor/common.py

```diff
@@ -789,9 +789,10 @@ def _webpage_read_content(self, urlh, url_or_request, video_id, note=None, errno
             self._downloader.to_screen(dump)
         if self.get_param('write_pages', False):
             basen = '%s_%s' % (video_id, urlh.geturl())
-            if len(basen) > 240:
+            trim_length = self._downloader.params.get('trim_file_name', 240)
```

Comment:
the param can be `None`
```suggestion
            trim_length = self._downloader.params.get('trim_file_name') or 240
```

Reply:
https://github.com/yt-dlp/yt-dlp/blob/0d32e124c69c5d53eb9dd286aa6147ec4cf52e54/yt_dlp/YoutubeDL.py#L475-L478
https://docs.python.org/3.6/library/stdtypes.html#dict.get
∴ not necesary

Reply:
```py
>>> x = {'a': None}
>>> print(x.get('a', 42))
None
>>> x.get('a') or 42
42
>>> x.get('b', 42)
42
>>> x.get('b') or 42
42
```

Reply:
It can't be `None`, but `0`, which might be a good reason to accept:
https://github.com/yt-dlp/yt-dlp/blob/0d32e124c69c5d53eb9dd286aa6147ec4cf52e54/yt_dlp/options.py#L1005-L1008

Reply:
Oh that makes sense then, wasn't sure.